osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/security/RsaEncryptionService.java

```diff
@@ -329,7 +330,7 @@ private static KeyFactory getKeyFactory() {
      * @return a randomly initialized {@value #ALGORITHM} key pair for the given
      *         {@code keysize}.
      */
-    public static KeyPair createKeyPair(final int keysize) {
+    public static KeyPair createKeyPair(final int keysize) throws EncrypterException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.opensmartgridplatform.shared.exceptionhandling.EncrypterException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)